model/src/test/kotlin/config/PackageConfigurationTest.kt

```diff
@@ -159,6 +160,45 @@ class PackageConfigurationTest : WordSpec({
             ) shouldBe true
         }
 
+        "return true if artifact source code origin and identifier are equal, and version is in range" {
+            val config =
+                PackageConfiguration(
+                    id = Identifier.EMPTY.copy(name = "some-name", version = "[51.0.0,60.0.0]"),
+                    sourceCodeOrigin = SourceCodeOrigin.ARTIFACT
+                )
+
+            config.matches(
+                config.id.copy(version = "55"),
+                ARTIFACT_PROVENANCE
+            ) shouldBe true
+        }
+
+        "return false if artifact source code origin and identifier are not equal, and version is in range" {
```

Comment:
I propose to drop the tests from 176 - 201, because testing all combinations becomes excessive.

The existing test `"return false if only source code origin is not equal"` in my view is sufficient.